### PR TITLE
Cleanup and fix Containerfile.nocache files

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@mrguitar @sallyom @vrothberg @Gregory-Pereira PTAL

File: recipes/natural_language_processing/codegen/bootc/Containerfile.nocache [hunk 1]
```diff
@@ -1,35 +1,22 @@
 # Example: an AI powered sample application is embedded as a systemd service
 # via Podman quadlet files in /usr/share/containers/systemd
 #
-# Use build command:
-# podman build --build-arg "sshpubkey=$(cat $HOME/.ssh/id_rsa.pub)" -t quay.io/exampleos/myos .
-# The --build-arg "SSHPUBKEY=$(cat ~/.ssh/id_rsa.pub)" option inserts your
-# public key into the image, allowing root access via ssh.
+# from recipes/natural_language_processing/summarizer, run
```
Comment (sallyom) [inline recipes/natural_language_processing/codegen/bootc/Containerfile.nocache:4]: summarizer->codegen

Comment (sallyom) [reply recipes/natural_language_processing/codegen/bootc/Containerfile.nocache:4]: (can fix w/ follow-up though)

**Review by sallyom — APPROVED** [review]
LGTM, will fix small nit in follow-up

**Pull request merged**